multicluster/apis/multicluster/v1alpha1/gatewaynode_types.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2021 Antrea Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GatewayNodeInfo includes basic information of Gateway Node
+type GatewayNodeInfo struct {
+	// The name of Gateway Node
+	Name string `json:"name,omitempty"`
+	// The Gateway IP of the Gateway Node, it might be assigned by user manually
+	// through annotation.
+	GatewayIP string `json:"gatewayIP,omitempty"`
+}
+
+// +genclient
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+
+// GatewayNode is the Schema for the gatewaynodes API
+type GatewayNode struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// The Gateway IP of the Gateway Node, it might be assigned by user manually
+	// through annotation.
+	GatewayIP string `json:"gatewayIP,omitempty"`
+	// Internal IP of Gateway Node
+	InternalIP string `json:"internalIP,omitempty"`
+}
+
+// ClusterInfo defines the desired state of ClusterInfo
+type ClusterInfo struct {
+	// ClusterID of the member cluster.
+	ClusterID string `json:"clusterID,omitempty"`
+	// ServiceCIDR are the IP ranges used by Service ClusterIP.
+	ServiceCIDR string `json:"serviceCIDR,omitempty"`
```

Comment:
To support dual stack later, would you use a slice?

Reply:
Maybe we can add another field ServiceCIDRv6 in the future like we have in the antrea-agent.conf?